packages/aila/src/core/llm/OpenAIService.ts

```diff
@@ -87,4 +94,31 @@ export class OpenAIService implements LLMService {
 
     return newStream.getReader();
   }
+
+  async generateObject<T>({
```

Comment:
A new method that we can use for categorisation etc. that uses structured outputs